.github/workflows/release_tests.yml

```
@@ -0,0 +1,89 @@
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+
+      - name: Install System Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt install python3-dev swig libssl-dev portaudio19-dev libpulse-dev libfann-dev
+
+      - name: Install core repo dependencies
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2102:info:1:17: Ranges can only match single chars (mentioned due to duplicates)

(shellcheck)